### PR TITLE
Add default value to cli flag

### LINKED ISSUE #613: Fix CLI confusing default value
The string `(default: 0)`, which is not in the source code, is added to the end of subcommand description. I guess its added by cli package.
```
$ fission env create -h
   --poolsize value                     Size of the pool, if not specified defaults to 3 (default: 0)
   --version value                      Environment API version: defaults to 1 (means v1 interface) (default: 0)
   --graceperiod value, --period value  The grace time for pod to perform connection draining before termination (optional, defaults to 360 seconds) (default: 0)
```

### PR DESCRIPTION
**Pull request by life1347**
Fix #613

---
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/619)

### CODE REVIEW

File: fission/main.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func main() {
 	specSaveFlag := cli.BoolFlag{Name: "spec", Usage: "Save to the spec directory instead of creating on cluster"}
 
 	// trigger method and url flags (used in function and route CLIs)
-	htMethodFlag := cli.StringFlag{Name: "method", Usage: "HTTP Method: GET|POST|PUT|DELETE|HEAD; defaults to GET"}
+	htMethodFlag := cli.StringFlag{Name: "method", Value: "GET", Usage: "HTTP Method: GET|POST|PUT|DELETE|HEAD"}
```
Comment (vishal-biyani) [inline fission/main.go]: @life1347 Since we are removing the default value message, is there any way for the user to know what the defaults are?

Comment (life1347) [reply fission/main.go]: If we assign value to `Value `  fields, the cli package will print default value at the end of command description automatically like following.
```
--method value               HTTP Method: GET|POST|PUT|DELETE|HEAD (default: "GET")
--executortype value         Executor type for execution; one of 'poolmgr', 'newdeploy' (default: "poolmgr")
--targetcpu value            Target average CPU usage percentage across pods for scaling (default: 80)

```

Comment (vishal-biyani) [reply fission/main.go]: Great!

**Comment by vishal-biyani** [comment]
LGTM

**Pull request merged**